pkgs/tools/package-management/cargo-outdated/default.nix

```diff
@@ -2,20 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-outdated";
-  version = "0.9.1";
+  version = "0.9.5";
 
   src = fetchFromGitHub {
     owner = "kbknapp";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "02gsarwm4gjkr9m4sfbjwp37xmqhch8qpyy027bxqkg8iyipxm69";
+    # This is the git commit that produced 0.9.5, according to crates.io, but
+    # the tag is missing in git. See here for details:
+    # https://github.com/kbknapp/cargo-outdated/issues/206
+    rev = "7685da3265749bb7ae2b436a132f51d19b409bff";
```

Comment:
To verify in the absense of https://github.com/kbknapp/cargo-outdated/issues/206, 

```
wget https://crates.io/api/v1/crates/cargo-outdated/0.9.5/download
tar -xzf download
cat cargo-outdated-0.9.5/.cargo_vcs_info.json
{
  "git": {
    "sha1": "7685da3265749bb7ae2b436a132f51d19b409bff"
  }
}
```

Reply:
I guess the maintainer simply forgot to push the tag? In any case, this looks good to me.

Reply:
I observed this from time to time with rust packages. Since the release is published at crates.io, we can use the git rev that points to this release as you did :+1: